feat(changelog): domains-and-dns-changed-pricing-update-for-some-tlds 2024-09-02 (#3657)
scaleway/docs-content: changelog/september2024/2024-09-02-domains-and-dns-changed-pricing-update-for-some-tlds.mdx

---
title: Pricing update for some TLDs
status: changed
author:

>     fullname: 'Join the #[xxx] channel on Slack.'

Reviewer: ```suggestion
    fullname: 'Join the #domains channel on Slack.'
```

    url: 'https://slack.scaleway.com'
date: 2024-09-02
category: network
product: domains-and-dns
---

> Starting Monday 2nd September beauty, college, hair, makeup, monster, quest, rent, skin will have a new pricing for classic actions (buy, renew, transfer).

Reviewer 2: ```suggestion
Starting Monday 2nd September, the following TLDs will have a new pricing for classic actions (buy, renew, transfer): .beauty, .college, .hair, .makeup, .monster, .quest, .rent, and .skin.
```

> Starting Monday 2nd September, the following TLDs will have a new pricing for classic actions (buy, renew, transfer): .beauty, .college, .hair, .makeup, .monster, .quest, .rent, and .skin.

Reviewer 3: ```suggestion
Starting Monday 2nd September 2024, a new pricing will apply to the following TLDs when buying, renewing, and  transferring domains: `.beauty`, `.college`, `.hair`, `.makeup`, `.monster`, `.quest`, `.rent`, and `.skin`.
```

> Starting Monday 9th September .com and .net will also be impacted.

Reviewer 2: ```suggestion
From Monday 9th September, .com and .net TLDs will also be impacted.
```

> From Monday 9th September, .com and .net TLDs will also be impacted.

Reviewer 3: ```suggestion
From Monday 9th September 2024, the pricing for `.com` and `.net` TLDs will also change.
```

> You can refer to this page to consult our new pricing : https://www.scaleway.com/en/domains-name/

Reviewer 3: ```suggestion
Refer to the [dedicated page](https://www.scaleway.com/en/domains-name/) to consult our new pricing.
```
